x-pack/plugins/spaces/index.ts

```diff
@@ -145,8 +145,17 @@ export const spaces = (kibana: any) =>
       registerUserProfileCapabilityFactory(async request => {
```

Comment:
I'm interesting to see how the user profile capabilities end up evolving with the granular app privs, because checking this stuff from spaces seems "wrong" if they're reaching into security, but let's just keep this in the back of our minds as the current approach works for the time being.

Reply:
Agree 💯